template/Gopkg.toml

```diff
@@ -1,9 +1,9 @@
 [[constraint]]
-  branch = "4.x"
+  version = "=1.0.0"
```

Comment:
should we release zenkit 4.0.0 first?

Reply:
I don't think so.  This template refers to zenkit, but the dependency is actually the other direction: zenkit uses the templae when we build a new service.